usig/sgx/usig-enclave.go

```diff
@@ -16,12 +16,69 @@
 
 package sgx
 
-// #cgo pkg-config: $SGX_URTS_PACKAGE
-// #cgo CFLAGS: -I${SRCDIR}/shim
-// #cgo LDFLAGS: -L${SRCDIR}/shim -lusig_shim
-//
-// #include <stdlib.h>
-// #include "usig.h"
+/*
+// TODO Use pkg-config?
+#cgo CPPFLAGS: -I/opt/intel/sgxsdk/include
```

Comment:
Obviously, hard-coding an absolute path is a bit hacky. This line seems to indicate that the cgo code still depends on SGX SDK...